### PR TITLE
GradleRIO and Visual Studio Code updates

### PR DESCRIPTION
**Pull request by KeithRieck**
I have updated the export files to generate robot projects that work with GradleRIO and that include support files for Visual Studio Code.

### CODE REVIEW

File: src/main/java/robotbuilder/exporters/ExportFile.java [hunk 1]
```diff
@@ -76,6 +86,10 @@ void backup(GenericExporter exporter) throws IOException {
         }
     }
 
+    private boolean equals(String s1, String s2) {
+        return (s1 == null) ? (s2 == null) : (s1.equals(s2));
+    }
```
Comment (JLLeitschuh) [inline src/main/java/robotbuilder/exporters/ExportFile.java]: There's already `Objects.equals` that does this. Why not use that?
https://docs.oracle.com/javase/7/docs/api/java/util/Objects.html#equals(java.lang.Object,%20java.lang.Object)

File: src/main/resources/export/cpp/vscode_launch.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
+{
+  // Use IntelliSense to learn about possible attributes.
+  // Hover to view descriptions of existing attributes.
+  // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
```
Comment (JLLeitschuh) [inline src/main/resources/export/cpp/vscode_launch.json]: JSON doesn't allow for comments in the RFC spec. Is this legal in this case?

File: src/main/java/robotbuilder/exporters/ExportFile.java (hunk 1 above)
Comment (KeithRieck) [reply src/main/java/robotbuilder/exporters/ExportFile.java]: OK, fixed.

File: src/main/resources/export/cpp/vscode_launch.json (hunk 2 above)
Comment (KeithRieck) [reply src/main/resources/export/cpp/vscode_launch.json]: Also fixed.  These files were generated by VS Code, so they will work. However, I have cleaned them up, and retested the results.

File: src/main/resources/export/cpp/files.yaml [hunk 3]
```diff
@@ -1,77 +1,105 @@
 ## Eclipse Project Files
```
Comment (AustinShalit) [inline src/main/resources/export/cpp/files.yaml]: Change to GradleRIO

File: src/main/resources/export/cpp/wpilib_preferences.json [hunk 4]
```diff
@@ -0,0 +1,6 @@
+{
+    "enableCppIntellisense": true,
+    "currentLanguage": "cpp",
+    "projectYear": "Beta2019",
+    "teamNumber": ${team-number}
+}
```
Comment (AustinShalit) [inline src/main/resources/export/cpp/wpilib_preferences.json]: EOL @ EOF

File: src/main/resources/export/java/wpilib_preferences.json [hunk 5]
```diff
@@ -0,0 +1,6 @@
+{
+    "enableCppIntellisense": false,
+    "currentLanguage": "java",
+    "projectYear": "Beta2019",
+    "teamNumber": ${team-number}
+}
```
Comment (AustinShalit) [inline src/main/resources/export/java/wpilib_preferences.json]: EOL @ EOF

**Comment by KeithRieck** [comment]
All of my changes are now accomplished by pull request #112

**Pull request closed without merging**